### ISSUE TITLE
Feature Request: add XLM - Stellar Lumens

### ISSUE THREAD

**Issue by paulpoco** [btdex/btdex#224]
Feature Request: replace XLA with XLM

XLA has not traded since May 2021.

XLM Stellar Lumens: 
- on 309 markets
- sending time is almost instant 
- low tx fee or no fee to send tx

**Comment by paulpoco** [comment]
XLM - Stellar Lumens needs in addition to the address, a memo for most exchanges 

https://developers.stellar.org/docs/glossary/transactions/

![Screenshot_20220430-110833_Chrome](https://user-images.githubusercontent.com/801414/166117455-bde0d998-fc83-44b4-ad3c-0cd5fdb31089.jpg)
![Screenshot_20220430-110354_Quick Trade](https://user-images.githubusercontent.com/801414/166117460-4eaaa631-5f09-4bab-aa08-7136c138de96.jpg)

**Comment by paulpoco** [comment]
Stellar uses [Ed25519 public/private keys](https://ed25519.cr.yp.to/). So you can research that if you want some more details.

If you're using a library (like the JS SDK) it's as simple as calling:

StrKey.isValidEd25519PublicKey(address);

If you want to see the details of how that's implemented, you can see here: [https://github.com/stellar/js-stellar-base/blob/master/src/strkey.js#L110](url)

In short: There is a version byte, a payload/data, and a checksum.
